test.disabled/ejabberd_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -242,6 +248,21 @@ generic_end_per_testcase(CaseName, Config) ->
 %% Service discovery test
 %%--------------------------------------------------------------------
 
+test_host_refreshing(_Config) ->
+    eventually_true("wait for refresher to converge trees states",
+                    fun() -> trees_for_connections_present() end, ?HOSTS_REFRESH_INTERVAL, 2),
+    ConnectionSups = out_connection_sups(asia_node),
+    [EuropeHost] = lists:filtermap(fun({europe_node1, Host, _}) -> {true, list_to_binary(Host)};
```

Comment:
Whoa, isn't it shooting mosquito with a cannon? Why can't we use `lists:keyfind/3`?